pelicanconf.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*- #
+from __future__ import unicode_literals
+
+AUTHORS = (
+    u'The Kinto team',
+)
+
+SITENAME = u'Kinto Storage'
+SITEURL = 'http://localhost:8000'
+
+PATH = 'content'
+
+TIMEZONE = 'Europe/Paris'
+
+DEFAULT_LANG = u'fr'
+
+DEFAULT_PAGINATION = False
+
+THEME = "theme"
+INDEX_SAVE_AS = 'news.html'
```

Comment:
@n1k0 we could use something like https://html5up.net/uploads/demos/telephasic/right-sidebar.html, what do you think?